build.sh

```diff
@@ -10,6 +10,29 @@ fi
 apk update
 apk add alpine-sdk util-linux strace file zlib-dev zlib-static autoconf automake libtool
 
+# Build static squashfuse
+apk add glib-dev  glib-static fuse-dev fuse-static zstd-dev zstd-static # fuse3-static fuse3-dev
+wget -c -q "https://github.com/vasi/squashfuse/releases/download/0.1.104/squashfuse-0.1.104.tar.gz"
+tar xf squashfuse-*.tar.gz
+cd squashfuse-*/
+./autogen.sh
+./configure --help
+./configure CFLAGS=-no-pie LDFLAGS=-static
+make -j$(nproc)
+make install
+/usr/bin/install -c -m 644 *.h '/usr/local/include/squashfuse' # ll.h
+cd -
+
+# Build static AppImage runtime
+# cd src/runtime # FIXME: We are "somewhere" in a container, so we can't access our own files?
+# FIXME: Crude workaround
```

Comment:
A container is like a chroot, you can't access files outside of it. You have to pass the paths that you want to be available, as shown here

https://github.com/probonopd/static-tools/blob/7bf023ff4853d45ae2492e2320438ae17a6ab222/.github/workflows/build.yaml#L51-L52

Reading this might help: https://github.com/marketplace/actions/run-on-architecture